src/appleseed.studio/mainwindow/project/disneymaterialcustomui.cpp

```diff
@@ -76,14 +75,14 @@ namespace
         }
     };
 
-    vector<Dictionary> collect_layer_definitions(const Dictionary& values)
+    std::std::vector<Dictionary> collect_layer_definitions(const Dictionary& values)
```

Comment:
Double qualifier.

Reply:
Oh, sorry. Interesting it compiles anyways...